website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -181,10 +181,10 @@ The height of the ListContainer is automatically determined based on the content
 A Header provides a fixed space at the top of the List. Typically, Headers house a search feature that allows the user to search/filter through the available options in the list. This is great for really long lists when filtering on complex datasets. 
 
 <div class="hds-dropdown__content">
-  <Doc::ListContainer class="hds-dropdown__list">
     <Hds::Dropdown::Header @hasDivider={{true}}>
       <Hds::Form::TextInput::Base @type="search" placeholder="Search" />
     </Hds::Dropdown::Header>
+    <Doc::ListContainer class="hds-dropdown__list">
```

Comment:
Nit: Could you fix the indenting for the Header above and this ListContainer wrapper?

Reply:
<img width="932" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/7607559c-6bd6-4fef-a01b-8b9063038e76">

Reply:
ok I think I found/resolved the issue.